packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md

```diff
@@ -0,0 +1,450 @@
+---
+title: Build a SCIM 2.0 server with entitlements
+meta:
+  - name: description
+    content: Use this guide to learn about the requirements to build a SCIM 2.0 server to support the exchange of entitlements between a downstream app and an Okta integration.
+layout: Guides
+---
+
+This guide teaches you the requirements to create a SCIM 2.0 server that supports provisioning entitlements to an app integration in Okta.
+
+---
+
+**Learning outcomes**
+
+* Learn the elements that are required to build a SCIM 2.0 server that supports provisioning entitlements to an app integration in Okta
```

Comment:
My understanding is that these are 'nutrition facts' that are part of the developer doc guides. I'm not sure how to better phrase it since they're the core of what the guide covers.

Reply:
Made an attempt to change it up to break up the monotony.